packages/aws_common/test/http/cancellation_test.dart

```diff
@@ -77,6 +79,14 @@ void main() {
           expect(response.body, emits(equals([1, 2, 3, 4, 5])));
         },
       );
+
+      test('can call cancel multiple times synchronously', () async {
+        final client_ = client();
+        final request = AWSHttpRequest.get(createUri('/body'));
+        final operation = client_.send(request);
+        unawaited(operation.operation.cancel());
+        await client_.close();
```

Comment:
nit/question: Should this have an `expect` like `await expectLater(client_.close(), completes);` ?

Reply:
Yeah, I can add it for clarity. Good thinking.